src/split_single_dyn_dim.cpp

```diff
@@ -68,37 +68,6 @@ has_one_dyn_dim(const std::unordered_map<std::string, shape>& param_shapes)
                                     dds_it->max};
 }
 
-namespace {
-struct find_static_2in_broadcasts
```

Comment:
Same as my other comment below in terms of not needing this matcher anymore like we don't need the testcase

Reply:
I moved these finders to a new pass `simplify_dynamic_ops`